lte/gateway/c/oai/tasks/s6a/s6a_task.c

```diff
@@ -85,8 +85,15 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
         OAILOG_DEBUG(
             LOG_S6A, "Sending s6a AIR for imsi=%s\n",
             received_message_p->ittiMsg.s6a_auth_info_req.imsi);
+        OAILOG_DEBUG(
```

Comment:
This log is not relevant to what we are logging.

Reply:
Resolved

Reply:
Remove debug log message

Reply:
Resolved